HLTrigger/Phase2/python/HLT_75e33/modules/pixelTracksHitDoublets_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 pixelTracksHitDoublets = cms.EDProducer("HitPairEDProducer",
-    clusterCheck = cms.InputTag(""),
+    clusterCheck = cms.InputTag("trackerClusterCheck"),
```

Comment:
I think this is not used, at the moment, for Phase 2 tracker. In the sense it has no effect (see [here](https://github.com/fwyzard/cmssw/blob/CMSSW_11_2_X/RecoTracker/TkSeedGenerator/python/trackerClusterCheck_cfi.py#L8)).

Reply:
I see.
Then I would prefer to keep this unset for the time being.